setup.py

```diff
@@ -108,92 +107,46 @@ def run(self):
         else:
             print("Stormpy - Warning: No support for POMDP analysis!")
 
-        # Set general cmake build options
         build_type = 'Debug' if self.config.get_as_bool("debug") else 'Release'
-        cmake_args = ['-DPYTHON_EXECUTABLE=' + sys.executable]
+        # Set cmake build options
+        print(self._extdir("core"))
```

Comment:
Is this print intended? If yes, could you add some text around it?

Reply:
Good catch. That is a left-over from some debugging. Will remove it.